<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GEMSeControl</class>
 <widget class="QDockWidget" name="GEMSeControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>170</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>170</height>
   </size>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFeatureMask</set>
  </property>
  <property name="windowTitle">
   <string>GEMSe Control</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="0" column="2" colspan="4">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>1</number>
       </property>
       <item row="0" column="1">
        <widget class="QPushButton" name="pbSamplingLoad">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Load Sample...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pbSample">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Sample...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pbClusteringCalc">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Cluster</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pbSelectHistograms">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Histograms...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pbClusteringStore">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Store Clusters...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pbClusteringLoad">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Load Clusters...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pbAllStore">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Store All</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="4" colspan="2">
     <widget class="QComboBox" name="cbColorThemes"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4" colspan="2">
     <widget class="QComboBox" name="cbRepresentative">
      <item>
       <property name="text">
        <string>Difference</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Average Entropy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Average Label</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Label Distribution</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="2" colspan="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Cluster View Preview Size</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Color Theme</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4" colspan="2">
     <widget class="QSpinBox" name="sbClusterViewPreviewSize">
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>75</number>
      </property>
      <property name="displayIntegerBase">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Operations</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Ref. Image</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="pbLoadRefImage">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="3">
     <widget class="QLineEdit" name="leRefImage">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="2" colspan="2">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Detail View Representative</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
